goal_src/jak1/engine/target/target-death.gc

```diff
@@ -144,8 +144,16 @@
       (set! *spawn-actors* #t)
       (set! *teleport* #t)
       (set! (-> *ACTOR-bank* birth-max) 1000)
-      (if  (-> *randomizer-settings* checkpoint-randomizer?)
-        (auto-save-command 'auto-save 0 0 *default-pool*)
```

Comment:
this was failing because the autosave from the cell pickup was running, and we only let 1 autosave proc run at a time